docs/source/search-runs.rst

```diff
@@ -97,7 +97,10 @@ You can search using two run attributes contained in :py:class:`mlflow.entities.
 
 .. code-block:: sql
 
-  attributes.artifact_uri
+  attributes.artifact_uri = 'models:/mymodel/1'
+  attributes.status = 'ACTIVE'
+  attributes.start_time >= 1664067852747
```

Comment:
Can you add some text explaining to users that the search terms here are in unix timestamp format?

Reply:
done